.gitignore

```diff
@@ -4,3 +4,4 @@ index.html
 coverage
 dist
 dist.tar
+.DS_Store
```

Comment:
This belongs in a [global `.gitignore`](http://islegend.com/development/setting-global-gitignore-mac-windows/)

Reply:
Not really. Sure, it is better to have it in a globsl gitignore file but this is often not the case. The same is with node-debug files, .idea directory and so on.

So far this change wys accepted in any project where I contributed to so it is still legit.